Makefile

```diff
@@ -12,18 +12,21 @@ install-node:
 install-hooks:
 	cp scripts/pre-commit .git/hooks/pre-commit
 
-lint:
+lint: copy-examples
 	npm run lint
 	poetry run flake8 **/*.py
 
 clean:
 	rm -rf build
 	rm -rf dist
 
-publish: clean
+publish: clean copy-examples
```

Comment:
Perhaps the "clean" target could remove the copied examples also?

Reply:
done